nri-redis/test/Spec/Settings.hs

```diff
@@ -0,0 +1,248 @@
+module Spec.Settings (tests) where
+
+import Database.Redis hiding (Ok)
+import qualified Dict
+import qualified Environment
+import qualified Expect
+import qualified Redis.Settings as Settings
+import qualified Test
+import Prelude (Either (..), Show (..))
+
+tests :: Test.Test
+tests =
+  Test.describe "parsing settings from environment"
+    [ decoderTests,
+      decoderWithEnvVarPrefixTests,
+      decoderWithCustomConnectionStringTests
+    ]
+
+-- we lean on `show` for equality since `Database.Redis.ConnectInfo` doesn't have
+-- an `Eq` instance (but does have a `Show` instance)
```

Comment:
💅🏼 May help to facilitate top-down scan of the file to move `expectEqualShow` right next to this comment and maybe have it be a func-doc. wdyt?

Reply:
resolved @ 3e7c07f